.travis.yml

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) 2013-2020 Arm Limited. All rights reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the License); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+language: sh
+os: linux
+dist: focal
+
+addons:
+  apt:
+    packages:
+      - cmake
+      - srecord
+
+matrix:
+  include:
+
+    # ARM_MUSCA_S1
+
+    - &compile-tests
+      stage: "Compile"
+      name: "Compile Regression and Compliance tests - release (ARM_MUSCA_S1)"
+      env: TARGET_NAME=ARM_MUSCA_S1 PROFILE=release
+      language: python
+      python: 3.8
+      install:
+        # Install arm-none-eabi-gcc
+        - pushd /home/travis/build && mkdir arm-gcc && cd arm-gcc
+        - curl -L0 "https://developer.arm.com/-/media/Files/downloads/gnu-rm/9-2020q2/gcc-arm-none-eabi-9-2020-q2-update-x86_64-linux.tar.bz2?revision=05382cca-1721-44e1-ae19-1e7c3dc96118&la=en&hash=D7C9D18FCA2DD9F894FD9F3C3DC9228498FA281A" --output gcc-arm-none-eabi-9-2020-q2-update.tar.bz2
+        - tar xf gcc-arm-none-eabi-9-2020-q2-update.tar.bz2
+        - export PATH="$PATH:${PWD}/gcc-arm-none-eabi-9-2020-q2-update/bin"
```

Comment:
Got in CI:
```
[Build-TF-M] 18:00:01: CMake Error at cmake/Common/FindGNUARM.cmake:156 (message):.
[Build-TF-M] 18:00:01:   ERROR: Unsupported GNUARM compiler version found on PATH..
```
This version is the one used in other project. I need to check if we need to downgrade the toolchain to a 2019 version for TF-M...

Reply:
Yep. We need to downgrade the GCC version to what we support... The toolchain used in our other repos are too new...

Reply:
Done